README.md

```diff
@@ -57,6 +58,7 @@ module private_gitlab {
 | backups\_enabled | Enabled or not the automated backups | `bool` | `false` | no |
 | retention\_days | Retention in days for automated backups | `number` | `null` | no | 
 | gitlab\_snapshot\_id | Snapshot id to use for restoring an existitent Gitlab | `string` | `null` | no |
+| swap_volume_size | Size in gb of the swap volume | `string` | `""` | yes |
```

Comment:
You have to escape underscore to don't be interpretated by MarkDown, like this:
`swap\_volume\_size`